pkg/operation/botanist/component/etcd/bootstrap_test.go

```diff
@@ -172,6 +219,7 @@ rules:
   - get
   - update
   - patch
+  - list
```

Comment:
Should we make changes here according to the `bootstrap.go` ?

Reply:
All changes have been replicated, unit tests are passing.

**Side note:** @rfranzke These `boostrap.go` and `bootstrap_test.go` are a real **pain** to maintain with each change to `etcd-druid`. Since I was recenlty working on several different PRs depending on changes in `etcd-druid`, I got rather sick of updating and fixing them. Is this the "improvement" we expected when replacing Helm charts with go code? At least for components that are part of gardener and are therefore changed / updated frequently, we should revisit this entire approach IMO.

Reply:
What exactly is the "pain" here? Can you detail this out a bit?